fixtures/shehds/12x12w-led.json

```
@@ -0,0 +1,117 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "12x12W LED",
+  "categories": ["Moving Head"],
+  "meta": {
+    "authors": ["Jason"],
+    "createDate": "2020-11-17",
+    "lastModifyDate": "2020-11-17"
+  },
+  "links": {
+    "productPage": [
+      "https://www.shehds.com/products/hot-led-beam-12x12w-rgbw-moving-head-lighting-dmx512-good-for-patry-dj-disco-concert"
+    ]
+  },
+  "physical": {
+    "dimensions": [25, 15, 31],
+    "weight": 4.6,
+    "power": 144,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED"
+    }
+  },
+  "availableChannels": {
+    "X axis rotation": {
+      "defaultValue": 1,
+      "highlightValue": 127,
+      "capability": {
+        "type": "Pan",
+        "angleStart": "0deg",
+        "angleEnd": "630deg"
+      }
```

```suggestion
    "Pan": {
      "fineChannelAliases": ["Pan fine"],
      "capability": {
        "type": "Pan",
        "angleStart": "0deg",
        "angleEnd": "540deg"
      }
```

The manual does use terms like "X axis rotation", but OFL generally uses terms like "Pan", and even other Shehds fixtures (like `led-spot-60w`) do that, so I'm making them consistent here.